src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "WebAssembly": {
+      "longName": "webassembly",
+      "shortName": "wasm"
+    },
+    "UWP": {
+      "longName": "uwp",
+      "shortName": "uwp"
+    },
+    "iOS": {
+      "longName": "ios",
+      "shortName": "ios"
+    },
+    "Android": {
+      "longName": "android",
+      "shortName": "android"
+    },
+    "macOS": {
+      "longName": "macos",
+      "shortName": "macos"
+    },
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5350297)